code/controllers/subsystems/payment.dm

```diff
@@ -1,3 +1,5 @@
+#define PASSENGER_FEE 50
```

Comment:
do we want this as a hard define or a var in the subsystem? making it a var opens the possibility for it to be changed midround in the future. @CPU-Blanc has done a lot with this code though so i'd want to get their take on it too.

Reply:
I want to have a good look through this as I've touched quite a bit of the payroll code.
Making the fee a var would be nice, as we could later hook the accounts console into it and allow the captain to up the price mid-round (for extra extortion)